Cannot find module 'path/posix'

After installing scribble (version 0.6.9) and trying to run it, I got the following error: `Error: Cannot find module 'path/posix'`. However, I do have `path/posix` installed as a global package in my nodejs setup.

After inspecting `eth-scribble/dist/bin/scribble.js`, I noticed that the require statement for that module is incorrect. It currently states:

```
const posix_1 = require("path/posix");
```

whereas it should be

```
const posix_1 = require("path-posix");
```

After manually patching the require statement as indicated above, scribble worked as expected.

--- a/src/bin/scribble.ts
+++ b/src/bin/scribble.ts
@@ -1,7 +1,6 @@
 #!/usr/bin/env node
 import fse from "fs-extra";
-import { dirname, join, relative, resolve } from "path";
-import { normalize } from "path/posix";
+import { dirname, join, relative, resolve, normalize } from "path";
 import {
     assert,
     ASTContext,

--- a/src/spec-lang/tc/typecheck.ts
+++ b/src/spec-lang/tc/typecheck.ts
@@ -1560,15 +1560,10 @@ export function tcMemberAccess(expr: SMemberAccess, ctx: STypingCtx, typeEnv: Ty
                     if (expr.member === rawDecl.name) {
                         // rawDecl.vType is defined, as you can't put a `var x;` in a struct definition.
                         expr.defSite = rawDecl;
+
                         return astVarToTypeNode(rawDecl, baseLoc);
                     }
                 }
-
-                throw new SNoField(
-                    `Struct ${baseToT.name} doesn't have a field ${expr.member}`,
-                    expr,
-                    expr.member
-                );
             }
         }
     }

